packages/liferay-npm-scripts/src/utils/spawnSync.js

```diff
@@ -27,4 +31,20 @@ module.exports = function(command, args = [], options = {}) {
 		stdio: 'inherit',
 		...options
 	});
+
+	if (status) {
+		throw new Error(
```

Comment:
In the case of `status`, I was wondering... are we piping the spawned process stderr and stdout to the main one? Otherwise, we might be missing not just the exit code, but valuable information.

Reply:
It says `stdio: 'inherit'` so it should be output to the console.

Reply:
So, from this, we should be seeing the errors in the build console, but just not signaling it properly? Just wondering about the EMF issue you're running into...

Reply:
Yes. The EMFILE is seen in the console, in fact. The problem is that because it doesn't stop the build, nobody notices it.

Reply:
Just as an example, if you look in my test plan above you can see what happened when I ran `../../node_modules/.bin/liferay-npm-scripts test` in the console with a conflicting jest config (you could see the error output on the console).